solana-library/spl_token.sol

```diff
@@ -176,4 +176,62 @@ library SplToken {
 
 		revert("account missing");
 	}
+
+	// This struct is the return of 'get_token_account_data'
+	struct TokenAccountData {
+		address owner;
+		address mintAccount;
+		uint64 balance;
+	}
```

Comment:
This is for later, but since you've deserialized everything in the mint, you'll probably want everything from here too.